api-reference/_hidden/dxFileManagerToolbar/items/items.md

```diff
@@ -11,8 +11,27 @@ Configures toolbar items' settings.
 
 ---
 
-The **FileManager** widget allows you to add default and create custom toolbar items.
+The **FileManager** widget allows you to add default and custom toolbar items.
 
+**Predefined Items**
+
+Predefined toolbar items include:
+
+- **'showNavPane'** - Shows or hides the navigation panel.  
+- **'create'** - Creates a new directory.
+- **'upload'** - Uploads a file.
+- **'refresh'** - Refreshes the file manager content and shows the progress panel.
+- **'download'** - Downloads a file.
+- **'move'** - Moves files and directories.
+- **'copy'** - Copies files and directories.
+- **'rename'** - Renames files and directories.
+- **'delete'** - Deletes files and directories.
+- **'switchView'** - Switches between the 'Details' and 'Thumbnails' file system representation modes.
+- **'clearSelection'** - Clears selection from files and directories in the Item View area.
+
+To add a predefined item to the toolbar, specify its [name](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#name) and optional settings ('visible', 'location', 'locateInMenu', 'text', 'icon', 'disabled') to the [items](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/) (for all items) or [fileSelectionItems](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/fileSelectionItems/) (for selected items) collections.
```

Comment:
How can you "specify something to" something else? Why do you use this preposition?

Reply:
updated